.photofinish.toml

```diff
@@ -202,3 +202,93 @@ files = [
 files = [
     "./test/fixtures/scenarios/host-details/9cd46919-5f19-59aa-993e-cf3736c71053_cloud_discovery_unknown.json",
 ]
+
+[host-vmdrbddev01-restore]
+
+files = [
+    "./test/fixtures/scenarios/healthy-27-node-SAP-cluster/240f96b1-8d26-53b7-9e99-ffb0f2e735bf_host_discovery.json",
```

Comment:
We could maybe open an improvement ticket to photifinish, to accept wildcards so we don't need to repeat so many similar files